metadata-ingestion/src/datahub/ingestion/source/gcs/gcs_source.py

```diff
@@ -88,7 +89,10 @@ def __init__(self, config: GCSSourceConfig, ctx: PipelineContext):
         super().__init__(config, ctx)
         self.config = config
         self.report = GCSSourceReport()
-        self.s3_source = self.create_equivalent_s3_source(ctx)
+        self.platform: str = "gcs"
```

Comment:
I think this constant is available

```suggestion
        self.platform: str = PLATFORM_GCS
```

Reply:
Changed, thank you!